src/vellum/workflows/events/types.py

```diff
@@ -43,10 +46,48 @@ def default_serializer(obj: Any) -> Any:
         )
     )
 
+class BaseParentContext(UniversalBaseModel):
+    span_id: UUID
+    parent: Optional['ParentContext'] = None
+
+
+class DeploymentParentContext(BaseParentContext):
+    deployment_id: UUID
+    deployment_name: str
+    deployment_history_item_id: UUID
+    workflow_version_id: UUID
```

Comment:
[oos] I suspect that we may want to make this field agnostic to prompt or workflow, but am still not sure what to call it. Let's defer until we have to cross that road with execute prompt

Reply:
Gonna actually raise in sdk channel now in the meantime